src/out_object.js

```diff
@@ -1,17 +1,31 @@
 // @flow
 import Phantom from './phantom';
-import crypto from 'crypto';
 
 export default class OutObject {
     _phantom: Phantom;
     target: string;
 
     constructor(phantom:Phantom) {
         this._phantom = phantom;
-        this.target = 'OutObject$' + crypto.randomBytes(16).toString('hex');
+        this.target = 'OutObject$' + randId(16);
     }
 
     property(name: string) {
         return this._phantom.execute(this.target, 'property', [name]);
     }
 }
+
+/**
+ * Generate an ascii random ID
+ * 
+ * @param  {Number} bytes number of bytes the ID should contain
+ * @return {String} a textual ID of `bytes` entropy
+ */
+function randId(bytes) {
```

Comment:
This again duplicated here? Feels more fitting if you create a file `util/random_id.js` and export it to be used here?